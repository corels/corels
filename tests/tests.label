{label=0} 0 0 1 1 0
{label=1} 1 1 0 0 1
