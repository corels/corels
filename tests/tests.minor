{group_minority} 0 0 0 0 0
