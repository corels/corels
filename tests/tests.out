{rule1} 1 0 0 1 1
{rule2} 1 0 1 0 1
{rule3} 0 1 1 0 1
{rule4} 0 0 1 1 0
{rul35} 0 0 0 1 1
